docs/contributor_guide/seed_scenarios.rst

```diff
@@ -0,0 +1,50 @@
+
+Seed ideas for scenarios:
+"""""""""""""""""""""""""
+These scenarios are seed ideas for thinking through ways to approach fairness questions, and considering what may be similar or different in your own problem context.  They may be starting points for coming up with new ideas for example notebooks, or some may require other approaches that are out of scope for *fairlearn*.
+
+
+Identifying potential tax fraud
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You're a member of an analytics team in a European country, and brought in to consult about a project that has already started to scale the deployment of models for predicting which tax returns may require further investigation for fraud.  The team has used a model trained in other jurisdictions by a large predictive analytics supplier, and hopes that they can leverage this at a lower cost that would be required to invest in the capability in-house.
+`Veale et al. (2018) <https://arxiv.org/pdf/1802.01029.pdf>`_
+
+Credit card fraud investigation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You're a data scientist at a financial services company, and your manager asks you to join an existing team.  This team has deploy a model trained on historical transaction data and now new data is arriving.  For each new transaction, the model predicts whether it is potentially fraudulent and then will trigger an alert and inspection by human analysts.  The output that matters for the company is the final decision by the human analyst of whether to block the transaction, allow it but flag for further investigation by anothe team, or flag the transaction as normal.
+`Weerts et al. (2019) <https://arxiv.org/abs/1907.03334>`_
```

Comment:
@hildeweerts might have some valuable insight on this one :)

Reply:
I have many opinions on this one! 

It's interesting you've titled this idea 'credit card fraud investigation'. In the Netherlands, we have a big debit card culture (e.g. the only reason I own a credit card is for traveling outside of Europe). This also implies that in the context of this paper, if the bank fails to catch a transaction of a victim of e.g. a phishing scheme, there's no way for the client to get their money back. This increases the impact of false negatives compared to a credit card scenario in which transactions are a bit less permanent. From the perspective of the bank though, the main reason for doing fraud detection of transactions is to avoid a fine for not trying hard enough. Basically the number of true positives should be high enough to pass regulations. At the same time, the number of false positives should be low enough such that (1) human analysts are able to process all positives (2) clients are not disrupted too much - any transaction flagged by the algorithm is put on hold, which can be quite disruptive if you're trying to buy a house for example.

Unfortunately, they didn't let me do any fairness related analyses of the system, at that point explainability was a more pressing concern for the bank. But I do think potential disparities in FP/FN are relevant. Although even more so for detecting money laundering schemes, which is more susceptible to undesirable forms of profiling - and much harder to do in general, because money launderers typically do not call you to complain about undetected fraud. 

In this particular case there were actually two different types of models; a machine learning model and a manually crafted rule-based system. I think it would be super interesting to compare both systems in terms of disparities. But that would probably be too complex for an example notebook ;)

PS. This brainstorm might be difficult to find for people who weren't tagged. I'm not sure what a better place would be though.

Reply:
@hildeweerts Awesome, thank you for sharing all this!  👍  It's so helpful if people want to explore whether fairlearn aspires to reach this kind of deployment context or not.

I think your descriptions here are a particularly great illustration of why the deployment context shapes how people think about trading off false positive and false negatives in a way that's far more complex than just picking which (accuracy, fairness) point fits them best.

I'm glad you pointed out the debit card difference that I missed, and it's great to highlight or remove US-centric assumptions like mine here.  I updated the example to be a Dutch company.  I tried to roll what you said into the paragraph here, and thanks so much for helping and sharing your expertise! 😄

Reply:
I think that all the scenarios below could use a sentence or two about who gets hurt by the false positives and false negatives.